libs/driver-adapters/executor/src/qc-test-worker/worker.ts

```diff
@@ -1,228 +1,223 @@
-            return initializeSchema(msg.params, msg.env)
-        case 'query':
-            return query(msg.params, unwrapState(), logs)
-        case 'startTx':
-            return startTransaction(msg.params.options, unwrapState())
-        case 'commitTx':
-            return commitTransaction(msg.params.txId, unwrapState())
-        case 'rollbackTx':
-            return rollbackTransaction(msg.params.txId, unwrapState())
-        case 'teardown':
-            return teardown(unwrapState())
-        case 'getLogs':
-            return logs
-        default:
-            assertNever(
-                msg,
-                `Unknown message type: \`${(msg as { type: unknown }).type}\``,
-            )
-    }
+  switch (msg.type) {
+    case 'initializeSchema':
+      return initializeSchema(msg.params, msg.env)
+    case 'query':
+      return query(msg.params, unwrapState(), logs)
+    case 'startTx':
+      return startTransaction(msg.params.options, unwrapState())
+    case 'commitTx':
+      return commitTransaction(msg.params.txId, unwrapState())
+    case 'rollbackTx':
+      return rollbackTransaction(msg.params.txId, unwrapState())
+    case 'teardown':
+      return teardown(unwrapState())
+    case 'getLogs':
+      return logs
+    default:
+      assertNever(
+        msg,
+        `Unknown message type: \`${(msg as { type: unknown }).type}\``,
+      )
+  }
 }
 
 function unwrapState(): State {
-    if (state === undefined) {
-        throw new Error('State is not initialized, call `initializeSchema` first')
-    }
-    return state
+  if (state === undefined) {
+    throw new Error('State is not initialized, call `initializeSchema` first')
+  }
+  return state
 }
 
 async function initializeSchema(
-    params: InitializeSchemaParams,
-    env: Env,
+  params: InitializeSchemaParams,
+  env: Env,
 ): Promise<ConnectionInfo> {
-    const {url, schema, migrationScript} = params
-
-    const driverAdapterManager = await setupDriverAdaptersManager(
-        env,
-        migrationScript,
-    )
-
-    const {compiler, adapter} = await initQueryCompiler({
-        url,
-        driverAdapterManager,
-        schema,
-    })
-
-    const transactionManager = new TransactionManager({
-        driverAdapter: adapter,
-    })
-
-    state = {
-        compiler,
-        driverAdapterManager,
-        driverAdapter: adapter,
-        transactionManager,
-    }
-
-    if (adapter.getConnectionInfo) {
-        return adapter.getConnectionInfo()
-    }
-
-    return {maxBindValues: undefined}
+  const { url, schema, migrationScript } = params
+
+  const driverAdapterManager = await setupDriverAdaptersManager(
+    env,
+    migrationScript,
+  )
+
+  const { compiler, adapter } = await initQueryCompiler({
+    url,
+    driverAdapterManager,
+    schema,
+  })
+
+  const transactionManager = new TransactionManager({
+    driverAdapter: adapter,
+    transactionOptions: {
+      maxWait: 30,
+      timeout: 30,
```

Comment:
Is 30 ms enough for all queries in tests?

Reply:
That's why the tests fail then. Thanks for the heads-up.

Where the unit is documented? Maybe we should name them `maxWaitMs` and `timeoutMs` to clarify.

Reply:
Modified the timeouts here to match the values in `TransactionManager.test.ts` in the `prisma` repo.

Reply:
Documented the units in `TransactionManager.test.ts` in the `prisma` repo.